src/styles/reset.css

```diff
@@ -0,0 +1,30 @@
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  border: 0;
```

Comment:
Do not reset margins and paddings using * selector. It is rather heavy operation to apply to every element.
Use tags for that.

Box-sizing, however, is fine with *